lib/phantomManager.js

```diff
@@ -116,9 +119,15 @@ PhantomManager.prototype._executeInWorker = function (worker, options, cb) {
     this._timeouts.push(timeout);
 
     worker.execute(options, function (err, result) {
-        if (err)
+        if (isDone)
```

Comment:
i was trying to preserve the original behaviour where if the `worker.execute` has an error the worker eventually will be recycled..

if you do: 

```js
worker.execute(options, function (err, result) {
   if (isDone) {
        return; 
   }

   isDone = true; //will cause the worker process won't be recycled in setTimeout
```

if the correct behaviour is only recycle if a timeout ocurred then `callbackWasCalled` is just duplicating `isDone`